nearcore/src/runtime/mod.rs

```diff
@@ -209,6 +211,7 @@ impl NightshadeRuntime {
             shard_tracker,
             genesis_state_roots: state_roots,
             migration_data: Arc::new(load_migration_data(&genesis.config.chain_id)),
+            num_epochs_to_keep_store_data,
```

Comment:
Since `NightshadeRuntime::new` is a public method it’s best to guard against this argument being too low. I’d consider making this `num_epochs_to_keep_tore_data.max(MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA)` or adding an assert that checks that the value is no less than the minimum.

Reply:
Added a max function.